src/config.ts

```diff
@@ -26,6 +26,8 @@ export async function getInputs(): Promise<Inputs> {
   const messageId = messageIdInput === '' ? 'add-pr-comment' : `add-pr-comment:${messageIdInput}`
   const messageInput = core.getInput('message', { required: false })
   const messagePath = core.getInput('message-path', { required: false })
+  const repoOwner = core.getInput('repo-owner', { required: false })
+  const repoName = core.getInput('repo-name', { required: false })
```

Comment:
```suggestion
  const repoOwner = core.getInput('repo-owner', { required: true })
  const repoName = core.getInput('repo-name', { required: true })
```

Since you set a default we can expect these to always be defined.